virttest/libvirt_xml/vm_xml.py

```diff
@@ -1852,7 +1888,7 @@ def __init__(self, virsh_instance=base.virsh):
                         'emulator_quota'):
                 accessors.XMLElementInt(slot, self, parent_xpath='/',
                                         tag_name=slot)
-        super(self.__class__, self).__init__(virsh_instance=virsh_instance)
+        super(VMCPUTuneXML, self).__init__(virsh_instance=virsh_instance)
```

Comment:
Is there any difference?

Reply:
seemingly this is required by inspekt check... The previous way will cause inspekt style check failed